packages/xo-web/src/xo-app/vm/tab-disks.js

```diff
@@ -248,6 +248,22 @@ const parseBootOrder = bootOrder => {
   return order
 }
 
+const WarningIllegalSr = connectStore({
+  hosts: createGetObjectsOfType('host'),
+})(({ hosts, requiredHost }) => {
+  const host = hosts[requiredHost]
+  return (
+    <span className='text-danger'>
+      <Icon icon='alarm' />{' '}
+      {host !== undefined
+        ? _('srNotOnSameHost', {
+            host: host.name_label,
+          })
+        : _('warningIllegalSr')}
```

Comment:
How would the user know how to fix this issue anyway? I think we can forget about the warning if the user can't see the host. Let me know if I'm missing something.